src/web/pages/App.tsx

```diff
@@ -1,27 +1,37 @@
-import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import HomePage from './HomePage';
+import ToolsLandingPage from './ToolsLandingPage';
+import FinOpsHubsPage from './FinOpsHubsPage';
+import FinOpsWorkbooksPage from './FinOpsWorkbooksPage';
+import GovernanceWorkbookPage from './GovernanceWorkbookPage';
+import AzureOptimizationEnginePage from './AzureOptimizationEnginePage';
+import PowerShellModulePage from './PowerShellModulePage';
+import BicepRegistryModulesPage from './BicepRegistryModulesPage';
+import OpenDataPage from './OpenDataPage';
+import PowerBIReportsPage from './PowerBIReportsPage';
+import CostOptimizationWorkbookPage from './CostOptimizationWorkbookPage';
 
-function App() {
 
+function App() {
   return (
-    <>
-      <head>
-        <h1>Kick start your FinOps efforts</h1>
-        <h2>Automate and extend the Microsoft Cloud with starter kits, scripts, and advanced solutions to accelerate your FinOps journey.</h2>
-      </head>
-      <p>
-          The Microsoft FinOps toolkit is an open-source collection of learning resources and customizable tools to help you adopt and implement FinOps capabilities that automate and extend the Microsoft Cloud. This includes:
-        </p>
-      <ul>
-        <li>Starter kits that help you get up and running.</li>
-        <li>Automation scripts to streamline cost configuration and management at scale.</li>
-        <li>Advanced solutions to facilitate building custom capabilities.</li>
-        <li>Learning resources and best practices about how to implement FinOps.</li>
-      </ul>
-      <p>
-        Read on to learn about the tools and roadmap for the FinOps toolkit or how to get involved.
-      </p>
-    </>
+    <Router>
+      <div>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/tools" element={<ToolsLandingPage />} />
+          <Route path="/tools/finops-hubs" element={<FinOpsHubsPage />} />
+          <Route path="/tools/power-bi-reports" element={<PowerBIReportsPage />} />
+          <Route path="/tools/cost-optimization-workbook" element={<CostOptimizationWorkbookPage />} />
+          <Route path="/tools/finops-workbooks" element={<FinOpsWorkbooksPage />} />
+          <Route path="/tools/governance-workbook" element={<GovernanceWorkbookPage />} />
+          <Route path="/tools/azure-optimization-engine" element={<AzureOptimizationEnginePage />} />
+          <Route path="/tools/powershell-module" element={<PowerShellModulePage />} />
+          <Route path="/tools/bicep-registry-modules" element={<BicepRegistryModulesPage />} />
+          <Route path="/tools/open-data" element={<OpenDataPage />} />
```

Comment:
Let's get rid of the extra layer, align them with the current paths, and probably sort them alphabetically for maintainability.
```suggestion
          <Route path="/bicep-registry" element={<BicepRegistryModulesPage />} />
          <Route path="/hubs" element={<FinOpsHubsPage />} />
          <Route path="/open-data" element={<OpenDataPage />} />
          <Route path="/optimization-engine" element={<AzureOptimizationEnginePage />} />
          <Route path="/power-bi" element={<PowerBIReportsPage />} />
          <Route path="/powershell" element={<PowerShellModulePage />} />
          <Route path="/workbooks" element={<FinOpsWorkbooksPage />} />
          <Route path="/workbooks/governance" element={<GovernanceWorkbookPage />} />
          <Route path="/workbooks/optimization" element={<CostOptimizationWorkbookPage />} />
```

Reply:
ok